package.json

```diff
@@ -69,8 +71,6 @@
     "files": [
       "dist/",
       "node_modules/**/*",
-      "app/renderer/index.html",
-      "app/renderer/images/",
```

Comment:
Images in index.html get built to `dist/` now... everything in `/app` can be ignored.